tests/unit/routes/reminders-test.js

```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
```

Comment:
Routes get tested by default in acceptance tests with lines like `visit('/')` or `currentURL('/')` etc. We dont need this file.